config/locales/simple_form.en.yml

```diff
@@ -0,0 +1,46 @@
+en:
+  simple_form:
+    "yes": 'Yes'
+    "no": 'No'
+    required:
+      text: 'required'
+      mark: '*'
+      # You can uncomment the line below if you need to overwrite the whole required html.
+      # When using html, text and mark won't be used.
+      # html: '<abbr title="required">*</abbr>'
+    error_notification:
+      default_message: "Please review the problems below:"
+    labels:
+      defaults:
+        first_name: "First Name"
```

Comment:
If you want to easily change the form labels generated by `SimpleForm`, this is where you'll need to do that.